Travels/ViewController/Trips/TripBudget/TripBudgetAssembly.swift

```diff
@@ -5,16 +5,15 @@
 //  Created by Anna on 03.06.2025.
 //
 
-import Foundation
 import UIKit
 
 enum TripBudgetAssembly {
     static func build(user: User?) -> UIViewController {
-        let view = TripBudgetViewController()
         let model = TripBudgetModel()
         let router = TripBudgetRouter()
-        let presenter = TripBudgetPresenter(view: view, model: model, router: router, currentUser: user)
-        view.presenter = presenter
+        let presenter = TripBudgetPresenter(model: model, router: router, currentUser: user)
+        let view = TripBudgetViewController(presenter: presenter)
+        presenter.attachView(view)
```

Comment:
а зачем сетить вью через метод, когда он у тебя не закрыт private? 
```suggestion
        presenter. view = view
```